.devcontainer/Dockerfile

```
@@ -0,0 +1,149 @@
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  libnss3 libatk-bridge2.0-0 libdrm2 libxkbcommon0 libgbm1 libasound2 \
+  libxcomposite1 libxrandr2 libxdamage1 libpango-1.0-0 libharfbuzz0b \
+  libgtk-3-0 libx11-xcb1 libxshmfence1 fonts-noto fonts-noto-cjk \
+  && rm -rf /var/lib/apt/lists/*
+# パッケージ導入後に Playwright とブラウザ
+WORKDIR /workspace/core
+RUN npm i -D playwright && \
+  npm install -g pnpm && \
+  pnpm exec playwright install --with-deps chromium
+
+RUN (type -p wget >/dev/null || (sudo apt update && sudo apt install wget -y)) \
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)